egs/wsj/s5/utils/mkgraph.sh

```diff
@@ -33,10 +32,11 @@ if [ $# != 3 ]; then
    echo "Usage: utils/mkgraph.sh [options] <lang-dir> <model-dir> <graphdir>"
    echo "e.g.: utils/mkgraph.sh data/lang_test exp/tri1/ exp/tri1/graph"
    echo " Options:"
-   echo " --mono          #  For monophone models."
-   echo " --quinphone     #  For models with 5-phone context (3 is default)"
-   echo " --left-biphone  #  For left biphone models"
-   echo "For other accepted options, see top of script."
+   echo " --remove-oov       #  For remove out-of-vocabulary."
```

Comment:
Replace 'For remove out-of-vocabulary' with the following text, suitably line-broken:
 If true, any paths containing the OOV symbol (obtained from oov.int in the lang directory) are removed from the graph G.fst during compilation.